packages/thirdweb/src/extensions/erc20/write/transferBatch.ts

```diff
@@ -0,0 +1,74 @@
+import type { BaseTransactionOptions } from "../../../transaction/types.js";
+import type { Prettify } from "../../../utils/type-utils.js";
+import { toUnits } from "../../../utils/units.js";
+import { multicall } from "../../common/__generated__/IMulticall/write/multicall.js";
+import { encodeTransfer } from "../__generated__/IERC20/write/transfer.js";
+/**
+ * Represents the parameters for a batch transfer operation.
+ */
+export type TransferBatchParams = Prettify<{
+  batch: Array<
+    { to: string } & (
+      | {
+          amount: number | string;
+        }
+      | {
+          amountWei: bigint;
+        }
+    )
+  >;
+}>;
+
+/**
+ * Transfers a batch of ERC20 tokens from the sender's address to the specified recipient address.
+ * @param options - The options for the batch transfer transaction.
+ * @returns A promise that resolves to the prepared transaction.
+ * @extension ERC20
+ * @example
+ * ```ts
+ * import { transferBatch } from "thirdweb/extensions/erc20";
+ * import { sendTransaction } from "thirdweb";
+ *
+ * const transaction = transferBatch({
+ *  contract,
+ *  batch: [
+ *    {
+ *      to: "0x...",
+ *      amount: 100,
+ *    },
```

Comment:
Maybe add another recipient in the example?

Reply:
mhk